src/validation.h

```diff
@@ -330,6 +330,9 @@ std::string FormatStateMessage(const CValidationState &state);
 /** Get the BIP9 state for a given deployment at the current tip. */
 ThresholdState VersionBitsTipState(const Consensus::Params& params, Consensus::DeploymentPos pos);
 
+/** Get the numerical statistics for the BIP9 state for a given deployment at the current tip. */
+BIP9Stats VersionBitsTipStatistics(const Consensus::Params& params, Consensus::DeploymentPos pos);
```

Comment:
Nit: I keep getting this and `VersionBitsStatistics` mixed up when I read through the changes. I think it would help readability to name this e.g. `CurrentVersionBitsStatistics` or `ActiveVersionBitsStatistics` or something.